### PR TITLE
Add slack-filter-channels command

### PR DESCRIPTION
**Pull request by craigsands**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: none

## Description
Adds the ability to list and filter channels to act on results

## Screenshots
None

## Minimum version of Cortex XSOAR
- [ ] 5.5.0
- [ ] 6.0.0
- [x] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Review by yaakovpraisler — COMMENTED** [review]
Looking good - see my comments

File: Packs/Slack/ReleaseNotes/1_4_0.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+##### Slack v2
+- Added 1 command
+  - ***slack-filter-channels***
+- Upgraded the Docker image to: *demisto/python3:3.9.5.20958*.
```
[the remark this answers is not in the document]
Comment (craigsands) [reply Packs/Slack/ReleaseNotes/1_4_0.md]: I think I misunderstood this. I just removed it and kept what already exists.

[the remark this answers is not in the document]
Comment (yaakovpraisler) [reply Packs/Slack/ReleaseNotes/1_4_0.md]: But we do need to update the docker image, so please revert the removed docker image, and also update here in the release notes the right updated docker image.
For more information and to have the updated docker image, you can see the [build validation](https://app.circleci.com/pipelines/github/demisto/content/98836/workflows/f2405682-5385-4e0d-99b4-4e92120441e6/jobs/394138).

**Review by yaakovpraisler — COMMENTED** [review]
Let's schedule a demo for your new command.

**Review by yaakovpraisler — COMMENTED** [review]
Also please update the test playbook with your new command.

**Comment by craigsands** [comment]
Sure I'll look into adding to the test playbook. In terms of scheduling a demo, how would I do that?

File: Packs/Slack/Integrations/Slack/Slack.py [hunk 2]
```diff
@@ -1986,6 +2094,48 @@ def slack_get_integration_context():
...
+            'IsGeneral': conversation.get('is_general'),
+            'IsPrivate': conversation.get('is_private'),
+        }
+        command_results_list.append(CommandResults(
+            outputs_prefix='Slack.Channel',
+            outputs_key_field='ID',
+            outputs=result_conversation
+        ))
+
+    return_results(command_results_list)
```
Comment (yaakovpraisler) [inline Packs/Slack/Integrations/Slack/Slack.py:2136]: Please return one CommandResults for all channels. (You can do it by adding each `result_conversation` to the `command_results_list` and return `return_results(CommandResults(command_results_list))`).
Also, return the results in human-readable in the CommandResults for better visibility. (for more information - https://xsoar.pan.dev/docs/reference/api/common-server-python#commandresults)

**Review by yaakovpraisler — COMMENTED** [review]
The result returned is too large.. let's see if we can use some limitation if there is in the slack API, if not, we can add a limit argument to the command and give it some default.

**Review by yaakovpraisler — APPROVED** [review]

**Pull request merged**